### PR TITLE
Admin layout update

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature

## Description

This PR does 3 things (tiny-enough that I decided to combine them):

1. It updates header's UI (less clunky, better crayons usage)
2. It removes Popper JS library and Bootstrap's Select. First hasn't been used, and the latter was replaced with classic select.
3. Added brand colors. 

## QA Instructions, Screenshots, Recordings

Log in as admin and go to `/admin` - you should see slightly updated header and maybe some updated accent colors. Everything else should remain the same.

**Before:**

image

**After:**

image

### UI accessibility concerns?

No.

## Added/updated tests?

No.

## [Forem core team only] How will this change be communicated?

It won't.

### CODE REVIEW

File: app/views/layouts/admin.html.erb [hunk 1]
```diff
@@ -30,28 +29,33 @@
...
-      <div class="flex items-center h-100 ml-auto">
-        <a href="https://admin.forem.com/" class="crayons-btn crayons-btn--ghost-brand hidden mr-2 whitespace-nowrap s:block">
-          Forem Admin Guide
-        </a>
-
-        <a href="/" class="crayons-btn hidden mr-2 whitespace-nowrap s:block">
-          Go to <%= community_name %> home page
-        </a>
-      </div>
+      <nav class="flex items-center h-100 ml-auto">
```
Comment (nickytonline) [inline app/views/layouts/admin.html.erb:55]: Thanks for adding a `<nav />`

File: app/views/layouts/admin.html.erb [hunk 2]
```diff
@@ -30,28 +29,33 @@
...
+        <%= link_to "Help", "https://admin.forem.com/", class: "c-link c-link--block whitespace-nowrap" %>
+        <%= link_to "Visit page »", root_path, class: "c-link c-link--block whitespace-nowrap" %>
```
Comment (nickytonline) [inline app/views/layouts/admin.html.erb:57]: Refactor (blocking): It's not clear what page is being visited.

```suggestion
        <%= link_to "Visit home page »", root_path, class: "c-link c-link--block whitespace-nowrap" %>
```

**Review by nickytonline — CHANGES_REQUESTED** [review]
Thanks for the cleanup! 🧹

**Review by nickytonline — APPROVED** [review]
🚀

File: app/views/layouts/admin.html.erb (hunk 2 above)
Comment (ludwiczakpawel) [reply app/views/layouts/admin.html.erb]: Hmm I've made another label tweak so it now says just "Home page" - does that make sense to you? As we don't really have "Visit" part for the "Help" link which is next to this one... So I thought it makes sense to make it consistent.

File: app/views/layouts/admin.html.erb [hunk 3]
```diff
@@ -54,7 +54,7 @@
 
       <nav class="flex items-center h-100 ml-auto">
         <%= link_to "Help", "https://admin.forem.com/", class: "c-link c-link--block whitespace-nowrap" %>
-        <%= link_to "Visit home page »", root_path, class: "c-link c-link--block whitespace-nowrap" %>
+        <%= link_to "Home page", root_path, class: "c-link c-link--block whitespace-nowrap" %>
```
Comment (nickytonline) [inline app/views/layouts/admin.html.erb:57]: Yeah, that's even better than my suggestion. 😎

File: app/views/layouts/admin.html.erb (hunk 1 above)
Comment (aitchiss) [inline app/views/layouts/admin.html.erb:55]:
```suggestion
      <nav aria-label="header" class="flex items-center h-100 ml-auto">
```

now we have multiple `nav`s on the page, labelling them has a big impact on screen reader users being able to get to the right one easily. TBH I couldn't think of a better name than "header" which would be announced as "header navigation", but happy for it to be called something else

**Review by aitchiss — APPROVED** [review]
Looks great! I left a small a11y suggestion(typical, eh?)

File: app/views/layouts/admin.html.erb (hunk 1 above)
Comment (ludwiczakpawel) [reply app/views/layouts/admin.html.erb]: thank you!

i also can't think of a better name 🤷

**Pull request merged**